docker/pdo_client.dockerfile

```diff
@@ -68,16 +65,13 @@ COPY --chown=${UNAME}:${UNAME} tools/*.sh ./
 # build it!!!
 RUN /project/pdo/tools/build_client.sh
 
-ARG PDO_HOSTNAME
+ARG PDO_HOSTNAME=localhost
 ENV PDO_HOSTNAME=${PDO_HOSTNAME}
 
-ARG PDO_LEDGER_URL
-ENV PDO_LEDGER_URL=${PDO_LEDGER_URL}
-
-ARG PDO_LEDGER_KEY_ROOT=/project/pdo/xfer/ccf/keys/
-ENV PDO_LEDGER_KEY_ROOT=${PDO_LEDGER_KEY_ROOT}
-
-# the client is set up for interactive access; the environment can be
-# set up by source /project/pdo/tools/start_client.sh with the arguments
-# to build a new client environment or copy one from the xfer directory
+RUN \
+  echo 'echo "\
+NOTE: to setup environment, \"source /project/pdo/tools/start_client.sh\"\n\
+   with the arguments \"--mode build\" to build a new client environment (default mode)\n\
+   or \"--mode copy\" to copy one from the xfer directory"\
+' > /project/pdo/.bashrc
```

Comment:
just create a .bashrc file & put this in the file. i will not approve changes that involve creating scripts in the dockerfile.

Reply:
yeah, i was thinking about that but then didn't really like to put a .bashrc file, so i decided on this.  But then in copy i can change file name, so while but it as pdo_client.bashrc  .and then COPY it ...

Reply:
fixed ...